docs/index.md

```diff
@@ -86,6 +86,9 @@ Installing the latest prometheus plugin will create a config file (`~/.steampipe
 ```hcl
 connection "prometheus" {
   plugin = "prometheus"
+
+
+  # The address of your Prometheus (can also be set with the PROMETHEUS_ADDRESS environment variable.).
```

Comment:
```suggestion
  # The address of your Prometheus.
  # Can also be set with the PROMETHEUS_ADDRESS environment variable.
```

Reply:
Can you please add a new line under the `Configuration` section in docs/index.md as well?